code/modules/unit_tests/reagent_mod_expose.dm

```diff
@@ -11,7 +11,7 @@
 		target.health = 80
 
 /datum/unit_test/reagent_mob_expose/Run()
-	var/mob/living/carbon/human/human = allocate(/mob/living/carbon/human)
+	var/mob/living/carbon/human/human = allocate(/mob/living/carbon/human, pick(GLOB.prisonwarp))
```

Comment:
I'm confused, what's with the changes on this test?

Reply:
Spawns the mob somewhere that's not nullspace, so the unit test won't fail because of an oxygenless environment putting out the phlogiston fire now the floating animation doesn't delay `Life()` anymore.

Reply:
I'm uncomfortable using the non-testing space in unit tests, but I'll take the blame on not making it easier.

In any case, can you at least comment the intention?

Reply:
Sure.